qiskit_aer/backends/unitary_simulator.py

```diff
@@ -260,12 +264,18 @@ def _execute(self, qobj):
         Returns:
             dict: return a dictionary of results.
         """
-        # Make deepcopy so we don't modify the original qobj
-        qobj = copy.deepcopy(qobj)
```

Comment:
Oops. We still need deep copy if Aer accepts qobj. At least, this code deletion should be in #1724. I will revert this code change.